draft-ietf-ppm-dap.md

```diff
@@ -2878,33 +2948,140 @@ struct {
 computed above and `encrypted_aggregate_share.ciphertext` is the ciphertext
 `encrypted_agg_share` computed above.
 
-The Helper's handling of this request MUST be idempotent. That is, if multiple
-identical, valid `AggregateShareReq`s are received, they should all yield the
-same response.
```

Comment:
The request is now a PUT, which implies idempotence in HTTP, so we don't need this remark